docs/projects/rot-13/index.md

```diff
@@ -0,0 +1,64 @@
+---
+title: ROT-13 in Every Language
+layout: default
+date: 2018-11-20
+last-modified: 2018-11-20
+featured-image:
+tags: [rot-13]
+authors:
+---
+
+ROT13 is a letter substitution cipher where every letter is replaced by the
+letter 13 letters after it alphabetically and wrapping from `Z` to `A` if necessary.
+
+    ABCDEFGHIJKLMNOPQRSTUVWXYZ -> NOPQRSTUVWXYZABCDEFGHIJKLM
+
```

Comment:
You should add a paragraph about my favorite ROT13 property 😆 which is that strings return to themselves when they are encrypted again.

Reply:
Actually, I'm happy to do this.